.pre-commit-config.yaml

```diff
@@ -23,3 +23,8 @@ repos:
     rev: 3.0.6
     hooks:
       - id: djhtml
+
+  - repo: https://github.com/hakancelikdev/unimport
```

Comment:
Sick of getting bugged about redundant imports? :-)
Would be better to have a separate PR for this though so it's easier to review.

Reply:
sick of not being bugged about it 😂